src/tox_uv/_run_lock.py

```diff
@@ -90,6 +93,16 @@ def _setup_env(self) -> None:
     @property
     def environment_variables(self) -> dict[str, str]:
         env = super().environment_variables
+
+        # Backward compatibility for people switching from pip to uv which forgot
+        # that uv does not load their constraint files.
+        # https://github.com/astral-sh/uv/issues/10686
+        for pip_var in ("PIP_CONSTRAINT", "PIP_CONSTRAINTS"):
+            if pip_var in env:
+                logger.warning("Copied %s into UV_CONSTRAINT for compatibility with pip", pip_var)
+                env["UV_CONSTRAINT"] = env[pip_var]
+                break
```

Comment:
I would say, no to this feature. Users should change this and not us silently.

Reply:
It is high likely that users will encounter this and they will not find out until much later, after. It took me 2 months for discover that I was not using the lock files on multiple projects. Few ideas:

- Maybe we can just make it a runtime warning instead?
- How many users do we expect to have PIP_CONSTRAINTS defined, use tox-uv and expect similar behavior?

AFAIK, so far just installing `tox-uv` would enable someone using tox to make use of uv, without having the user forced to make any other modification to their tox.ini file. That is important especially for those that install the plugin at user/system level and decide to test another random python project.